waterbutler/providers/rushfiles/provider.py

```diff
@@ -182,12 +182,11 @@ async def delete(self,  # type: ignore
                             "DeviceId": "waterbutler "
                         }),
                         headers={'Content-Type': 'application/json'},
-                        expects=(200, 404,),
+                        expects=(200, 400, 404,),
                         throws=exceptions.DeleteError,
                     )
-        if response.status == 404:
+        if response.status == 400 and response.status == 404:
```

Comment:
You will never get 400 AND 404 at the same time. This will always return false.
```suggestion
        if response.status == 400 or response.status == 404:
```

Reply:
すみません、変更しました